README.md

```diff
@@ -1,9 +1,12 @@
 # Eclipse Cyclone DDS
 
-Eclipse Cyclone DDS is by far the most performant and robust DDS implementation available on the
-market. Moreover, Cyclone DDS is developed completely in the open as an Eclipse IoT project
+Eclipse Cyclone DDS is a very performant and robust open-source DDS implementation.  Cyclone DDS is developed completely in the open as an Eclipse IoT project
```

Comment:
I've dealt with too many people that try to convince others by playing fast and loose with the truth — there are less polite words that would perhaps better describe my sentiment regarding those methods and the people employing them, but, you know, children may be reading these comments! — and who would not be able to provide solid evidence for their claims. There have even been those that "cosmetically altered" performance figures they got from me personally (not related to Cyclone, should anyone be wondering), by simply doubling the figures to get something provably impossible, and who afterward explained that "no one ever believes the numbers anyway, so better inflate them first". It sickens me ... and that form of madness certainly did not get man onto the moon and back.

I wish I had _evidence_ of Cyclone being "the most" performant DDS implementation — but it is impossible for me to ever get that evidence, and that has nothing to do with the actual performance. The reasons are simply that there are vendors that do not allow performing and/or publishing reproducible measurements, and that occasionally one discovers other implementations one was not aware of in the first place. So the old claim is simply a boast.

There _is_ evidence that Cyclone has a very good performance in general, and there are good grounds to believe that its performance (as measured in data rate or latency) outclasses that of various other implementations, perhaps all. So "very" is a defensible, if unimaginitve, choice of word.

I'd rather people look at the numbers and decide whether it will do, than have them believe grandiose claims. And if they allow themselves to be taken in by such claims without questioning the evidence behind them, then they fully deserve the lesser quality they will end up with. There is only one way to excellence, and that is honesty.

😁